File: leet-code-problems/Two-sums/TwoSums.py
def two_sums(nums, target):
    for i in range(len(nums)):
        for j in range( 1,  len(nums)):
            if nums[i] + nums[j] == target:
                return [i, j]
    
output = two_sums([4,7,8,13], 15)
print(output)

File: leet-code-problems/Two-sums/TwoSums.js
const twoSums = (nums, target) => {
  for (let i = 0; i < nums.length; i++) {
    for (let j = 1; j < nums.length; j++) {
      if (nums[i] + nums[j] === target) {
        return [i, j];
      }else{
        console.log("couldnt find the no")
      }
    }
  }
};
const output = twoSums([8, 4, 13, 7], 15);
console.log(output);
